fetch.bs

```diff
@@ -810,6 +810,20 @@ the empty string,
 "<code>worker</code>", or
 "<code>xslt</code>". Unless stated otherwise it is the empty string.
 
+<p>A <dfn export id=concept-potential-destination>potential destination</dfn> is "<code>fetch</code>" or a <a for=request>destination</a>.
```

Comment:
We should also move this to its own section. Maybe just a new section after "Responses" called "Miscellaneous"?